readthedocs/projects/signals.py

```diff
@@ -34,5 +35,12 @@ def handle_project_import(sender, **kwargs):
             if service is not None:
                 if service.setup_webhook(project):
                     messages.success(request, _('Webhook activated'))
+                    _set = True
                 else:
                     messages.error(request, _('Webhook configuration failed'))
+    if not _set:
+        messages.error(
+            request,
+            _('No accounts available to set webhook on. '
+                'Please connect either a GitHub or BitBucket account.')
```

Comment:
Addressed the first two. Not sure what to do about the later. I think there are also cases where they have no accounts connected and are importing a public project, so this might be misleading as well.

I think we should probably be moving the webhook setup into the project import view/process, instead of having it be a signal?